config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
This is tested in the cakephp core actually. What version of cakephp are you running where this doesn't work?

Also, at least on my shell (bash) `env` prints the current env, it doesn't set an environment variable. What shell are you using?